Sources/ForTag.swift

```diff
@@ -6,9 +6,15 @@ class ForNode : NodeType {
   let nodes:[NodeType]
   let emptyNodes: [NodeType]
   let `where`: Expression?
+  let label: String?
 
   class func parse(_ parser:TokenParser, token:Token) throws -> NodeType {
-    let components = token.components()
+    var components = token.components()
+
+    var label: String? = nil
+    if components.first?.hasSuffix(":") == true {
```

Comment:
No need for `== true`

Reply:
Oh I see, no idea what the general style guide is then. Personally I use `?? false` in `return` statements, and sometimes in `if` statements as well.

Reply:
Right, so the consensus seems to be either:
- `if components.first?.hasSuffix(":") == true {` (yours)
- `if (components.first ?? "").hasSuffix(":")` (getting rid of the optional)
- `if let firstComponent = components.first, firstComponent.hasSuffix(":") {` (also getting rid of the optional)

Reply:
I meant to use `components[0].hasSuffix(:)`. There always will be at least 1 component, so it's safe. We can also check before that if there is more than one component because that should be the case with labels (`{% label: %}` is not a valid syntax)

Reply:
I'm not too big of a fan of using subscript without bounds checks, so I'd add at least an `.isEmpty` check. Or keep the current syntax, my original comment only applied because I didn't notice it was Optional.